hw2/05-form.js

```diff
@@ -1 +1,33 @@
-// Add your code here
+const form = document.querySelector("form"); // Get form
+form.addEventListener("submit", handleSubmit); // Get input on submit
+
+// Extract form information and output in a modal
+function handleSubmit(event) {
+  event.preventDefault();
+
+  // Extract info from form
```

Comment:
Somewhat of a personal preference here, but I would recommend either placing a blank line between the console log and the next variable definition or performing all the console logs after all the data is caught. As it stands, it is a bit of a wall of text and can be difficult to read

Reply:
Yeah.  I forgot to note that this is a WIP.  The final form.js will not have the console.logs at all. I'm struggling with this modal!